Sources/PowerAuth2ForExtensions/PowerAuthAuthentication.m

```diff
@@ -30,9 +30,6 @@ @implementation PowerAuthAuthentication
     NSInteger _objectUsage;
 }
 
-#define AUTH_FOR_COMMIT         1
```

Comment:
Removing the defines and using the numbers directly in code is kind of confusing what is going on.